src/xcode/ENA/ENA/Source/Scenes/Checkin/QRCodeScanner/CheckinQRCodeScannerViewModel.swift

```diff
@@ -68,6 +86,32 @@ final class CheckinQRCodeScannerViewModel: NSObject, AVCaptureMetadataOutputObje
 		captureSession?.stopRunning()
 	}
 
+	/// toggle torchMode between on / off after finish call optional completion handler
```

Comment:
I think this comment is not needed, because the information is readable from the func name.